src/bionics.cpp

```diff
@@ -1492,14 +1492,14 @@ bool player::install_bionics(const itype &type, int skill_level)
         add_msg(m_good, _("Successfully installed %s."), bionics[bioid].name.c_str());
         add_bionic(bioid);
 
-        if (bioid == "bio_ears") {
+        if( bioid == "bio_eye_optic" && g->u.has_trait( "HYPEROPIC" ) ) {
+            g->u.remove_mutation( "HYPEROPIC" );
+        } if( bioid == "bio_eye_optic" && g->u.has_trait( "MYOPIC" ) ) {
```

Comment:
formatting here is weird: keep '{' on one line and move the rest to another one.